features/cli/add_mfa_device.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Storing MFA device information
```

Comment:
Need to add the Gemalto hardware token scenario.
These are not an AWS ARN, but rather a GAKTxxxxx value.